sync/class.jetpack-sync-module-woocommerce.php

```diff
@@ -113,4 +118,115 @@ public function estimate_full_sync_actions( $config ) {
 	private function get_where_sql( $config ) {
 		return '1=1';
 	}
+
+	public function options_whitelist( $list ) {
```

Comment:
Optional, but it may be nice to add phpdoc function documentation to these functions -- explaining what list can be, etc.  If someone wants to just add a single value, maybe we should have a test in `add_to_whitelist` or something that just casts the values to arrays before the `array_merge()` ?